### ISSUE TITLE
Integrating the autotick bot docs with the conda-forge main docs

### ISSUE THREAD

**Issue by ForgottenProgramme** [conda-forge/conda-forge.github.io#1460]
Autotick bot docs: https://regro.github.io/cf-scripts/#

**Comment by viniciusdc** [comment]
I think we could summarize some of the most important features and a general workflow for the Bot environment inside the INfrastructure section of the docs, as @beckermr suggested. 

But, before that we should carefully inspect some of the information presented on those sections (blog), there are some outdated information that we need to update as well (like the [actions infrascture](https://regro.github.io/cf-scripts/github_actions_infrastructure.html#github-actions-infrastructure))

**Comment by beckermr** [comment]
Ahh I will leave notes here!

**Comment by beckermr** [comment]
OK notes on items to include before I forget.

maintainer documentation:

1. possible keys for the conda-forge.yaml (https://conda-forge.org/docs/maintainer/conda_forge_yml.html#bot)
2. bot rerun labels
3. conda-forge admin commands for adding bot-rerun label (https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-rerun-bot)
4. command for adding automerge (https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-bot-automerge)
5. rules about when PRs come
  a. for versions, if three version PRs are open, no more will be made
  b. bot detects versions only if using a src URL with a version jinja2 variable and the version is roughly compatible with semver
  c. for ABI migrations, the bot will generate the migration PR only once the upstream package has been built
  d. ABI migrations are closed once all of the PRs in the migration have been issued

**Comment by beckermr** [comment]
The stuff for the migration yaml docs is a lot less complete.

migration yaml docs:

1. some of the relevant keys are in the example here: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/example.exyaml - however this is not all of them
2. there are extra keys we should add - the pyp37 migration has most of them: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/pypy37.yaml
  a. paused: if True, the migration is not marked as done but no PRs will be issued and it will not appear on the status page. This is important for migrations you want to stop but are not merged into the innings. If these migrations were deleted, then any feedstocks that got them would be misconfigured if rerendered.
  b. longterm: if True, this migration will show on the status page in a separate section. This key is used to highlight big migrations that we expect to a long time like python minor version bumps, cuda, etc.
  c. pr_limit: indicates how many PRs a migration can generate per run of the bot. This can be used to slow or speed up certain migrations.

I think this si good for starters. We'll want more docs on the kinds of migrations maybe but that is a very advanced topic.
